posthog/session_recordings/sql/session_replay_event_sql.py

```diff
@@ -34,7 +35,8 @@
     size Int64,
     event_count Int64,
     message_count Int64,
-    snapshot_source LowCardinality(Nullable(String))
+    snapshot_source LowCardinality(Nullable(String)),
+    snapshot_library Nullable(String)
```

Comment:
from a clickhouse review perspective - I did start with this as a low cardinality string but CH is still very unhappy with mapping aggregate types of `LowCardinality(Nullable(String))` so I didn't fight it